openapi/publicapi_spec.json

```diff
@@ -227,7 +227,7 @@
             "type" : "string"
           }
         }, {
-          "description" : "To date of payments to be searched (this date is exclusive). Example=2015-08-14T12:35:00Z",
+          "description" : "To date of payments to be searched (this date is exclusive). Example=2015-08-14T12:35:00.000Z",
```

Comment:
As I stated in https://github.com/alphagov/pay-publicapi/pull/1575, I’m not so sure about this because I believe `2015-08-13T12:35:00Z` is a valid input format. Let’s wait to see if we get any feedback on the other PR.